### ISSUE TITLE
Expansion inside window-status-style don't work as expected.

### ISSUE THREAD

**Issue by cevhyruz** [tmux/tmux#3094]
### Issue description
I'm trying to change the background color of the window list depending on `pane_mode`,
the background should be orange if in copy/view-mode and black otherwise.

To reproduce, use this snippet:
```tmux
set-option -g window-status-current-style \
"bg=#{?#{m/r:(copy|view)-mode,#{pane_mode}},orange,black}"

set-option -g window-status-style \
"bg=#{?#{m/r:(copy|view)-mode,#{pane_mode}},orange,black}"
```
Using above will only change bg for the current window and not for the remaining windows

### Required information

* tmux version (`tmux -V`). tmux next-3.4 (master at 88d9a8fc)

**Comment by nicm** [comment]
Are you expecting pane_mode to always be the mode of the active pane in the
current window? Because it won't be.

You can probably do what you want using the W operator with ##{pane_mode}
for the current window and nothing for the others.

**Comment by nicm** [comment]
Yes it would, because you are presumably only changing mode on one window
not on all of them.

**Comment by nicm** [comment]
Your best bet will probably be to use the W operator to get the pane_mode
for the current window even when window-status-format is being expanded for
other windows.

**Comment by cevhyruz** [comment]
>Yes it would, because you are presumably only changing mode on one window
not on all of them.

ohh I get it now. I probably really need to use W operator to cycle through. 
Thanks for the fast reply
